src/core/ComboBoxModel.cpp

```diff
@@ -37,7 +36,7 @@ void ComboBoxModel::addItem( QString item, unique_ptr<PixmapLoader> loader )
 
 void ComboBoxModel::replaceItem(std::size_t index, QString item, unique_ptr<PixmapLoader> loader)
 {
-	assert(index < m_items.size());
+	Q_ASSERT(index < m_items.size());
```

Comment:
I believe we're slowly trying to remove Qt from the core, so I think a standard `assert` is actually better here.

Reply:
One more difference: `Q_ASSERT` doesn't get removed in release builds unless `QT_NO_DEBUG` is defined.

Reply:
Agreed, switching back to `assert` then.